aspnetcore/mvc/advanced/custom-model-binding/3.0sample/PolymorphicModelBinding/Pages/Shared/_Layout.cshtml

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>@ViewData["Title"] - PolymorphicModelBinding</title>
-    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
```

Comment:
The minified files don't exist in the repo, so the css and js are broken without this change.